src/Storage.java

```diff
@@ -0,0 +1,45 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+class Storage implements DataToFromComptue {
+    private String outName = "StreamOfIntsOut";
+    private String inName = "StreamOfIntsIn";
+    private int files = 0;
+
+    public Storage() {

```

Comment:
Empty method, I am assuming it is due to the implementation of the API to make it work. If not, what for?